### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/mermaid.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#647] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/mermaid.html

让你的 VuePress 站点中的 Markdown 文件支持 mermaid。

https://theme-hope.vuejs.press/zh/guide/markdown/mermaid.html

**Comment by vipdcd** [comment]
在使用时报dayjs does not provide an export named 'default'，如何解决？

**Reply by Mister-Hope** [reply]
先升级到最新版，如果可以复现，开 issue 给我。
